### DISCUSSION TITLE
Interactive Jupyter Widgets Requiring Python Process

### DISCUSSION THREAD

**Discussion by JeremyBYU** [quarto-dev/quarto-cli#4420] [Q&A] [answered]
I was trying to do something very similar to this [issue](https://github.com/quarto-dev/quarto-cli/issues/2693) of adding a python jupyter interact widget into my slides.

Is there any proposed feature for Quarto to add the option to render with:

1. A full running python process on a users machine. Basically running jupyter behind the scenes and dynamically updating slides/html
OR/AND
2. Embedding python and matplotlib in the created HTML as the link above to Shiny seems to be doing.

Just wondering if this is in the pipeline or if completely out of scope.

Thanks!

**Comment by cscheid** [comment, marked as answer]
Our Python interactive story will be through support for https://shiny.rstudio.com/py/, because its reactivity paradigm is much more closely aligned with Observable JS and the R shiny package. We will definitely do this, but with that said, it is not in our short-term (3-6 months) plans.

**Comment by JeremyBYU** [comment]
Thanks for clarification!
